src/components/GraphiQL/styles.ts

```diff
@@ -55,6 +57,9 @@ export const useEditorStyles = () => {
 };
 
 export const useDashboardTheme = () => {
+  const color = ComputedStyle.getPropertyValue("--mu-colors-background-plain");
```

Comment:
**suggestion**: we have a handy property in the `useTheme` hook that may help here

You can use `{ themeValues } = useTheme()` from `@saleor/macaw-ui/next` and get hsla value for `themeValues.colors.backgroundPlain`

An example usage can be found here: https://github.com/saleor/saleor-dashboard/blob/main/src/components/Datagrid/Datagrid.tsx#L123